src/bit-systems/media-loading.ts

```diff
@@ -176,8 +210,11 @@ function* loadAndAnimateMedia(world: HubsWorld, eid: EntityID, clearRollbacks: C
   removeComponent(world, MediaLoader, eid);
 
   if (media) {
+    const box = getBox(world, eid, media);
+    MediaContentBounds.set(media, box.getSize(new Vector3()));
```

Comment:
What happens if an exception is caught in `loadMedia()`? `media` is non-null but `MediaLoaded` component is not added to an entity. So, here `MediaContentBounds` is accessed even without `MediaLoaded` component.

Reply:
Right, it would be harmless but we don't need to add `MediaContentBounds` in that case.